style.css

```diff
@@ -482,21 +480,28 @@ textarea {
     background-color: #fff;
     border-bottom-left-radius: 10rem;
     margin-bottom: 6rem;
-
-    /* height: 100vh; */
   }
 
   .header-section {
     display: flex;
     flex-direction: column;
+    justify-content: center;
+    align-items: center;
     background: url(./images/desktop-background.png);
     background-size: cover;
     background-repeat: no-repeat;
     background-position: 100% 100%;
   }
```

Comment:
- [ ] [OPTIONAL] Please consider adding the correct margin or padding to the toolbar on the left and right side to match the Figma template. The red marker indicates that there should be about 155px space on both sides. 

Your work  
![image](https://user-images.githubusercontent.com/84156714/221356100-41add849-0a2e-43c8-9ac2-f729ae690166.png)

Figma Design
![image](https://user-images.githubusercontent.com/84156714/221356220-56051bb3-34bc-4911-bff9-af837db774cc.png)